brightray/common/application_info_win.cc

```diff
@@ -62,4 +64,45 @@ bool GetAppUserModelID(ScopedHString* app_id) {
   return app_id->success();
 }
 
+bool IsRunningInDesktopBridgeImpl() {
+  if (IsWindows8OrGreater()) {
+    // GetPackageFamilyName is not available on Windows 7
+    typedef HRESULT(WINAPI * GetPackageFamilyNameFuncPtr)(
```

Comment:
I would use `using GetPackageFamilyNameFuncPtr = decltype(&GetPackageFamilyName);` to avoid copying the type definition